src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-param-reassign */
```

Comment:
- Hey, please, let's try to write codes that conform to the linter as this would help improve our development skill. please, let's remove this eslint disable and fix the linter.